test/unit/player-play.test.js

```diff
@@ -98,3 +104,26 @@ QUnit.test('tech ready + has source + changing source = wait for loadstart', fun
   this.player.trigger('loadstart');
   assert.strictEqual(this.techPlayCallCount, 1, 'tech_.play was called');
 });
+
+QUnit.test('tech not ready +  currentTime =  wait until canplay event to apply currentTime value', function(assert) {
```

Comment:
I don't think that `player-play.test.js` is the right location for these tests. This file is around various tests cases around calling `play()`. Probably best to move it to `player.test.js`

Reply:
Done.